### PR TITLE
Contrib/issue 492

### PR DESCRIPTION
**Pull request by shaheemazmalmmd**
https://github.com/fossology/fossology/issues/492

### CODE REVIEW

File: src/nomos/agent/parse.c [hunk 1]
```diff
@@ -405,7 +405,7 @@ char *parseLicenses(char *filetext, int size, scanres_t *scp,
    */
   if (INFILE(_LT_MYSQL_EXCEPT) || INFILE(_PHR_FREE_LIBRE)) {
     if (INFILE(_TITLE_ALFRESCO)) {
-      INTERESTING("Alfresco/FLOSS");
+      INTERESTING("Alfresco.FLOSS");
```
Comment (steffen-weber) [inline src/nomos/agent/parse.c]: licenseref.sql contains 'Alfresco-FLOSS'

File: src/nomos/agent/parse.c [hunk 2]
```diff
@@ -5824,18 +5824,18 @@ char *parseLicenses(char *filetext, int size, scanres_t *scp,
    */
   if (INFILE(_LT_USGOVT_1)) {
     if (INFILE(_CR_URA)) {
-      MEDINTEREST("URA(gov't)");
+      MEDINTEREST("URA(govt)");
```
Comment (steffen-weber) [inline src/nomos/agent/parse.c]: here are still brackets

**Comment by steffen-weber** [comment]
The licenseref.sql contains 'Helix/RealNetworks-EULA' (maybe others). Hence
* update of licenseref.sql
* migration script for existing table license_ref

are desired.

**Comment by shaheemazmalmmd** [comment]
Closing the pull request as it is having a merge issue with master.

**Pull request closed without merging**